modules/apps/item-selector/item-selector-taglib/src/test/java/com/liferay/item/selector/taglib/internal/display/context/GroupSelectorDisplayContextTest.java

```diff
@@ -48,15 +53,25 @@ public class GroupSelectorDisplayContextTest {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
-	@Before
-	public void setUp() {
+	@BeforeClass
```

Comment:
@MrJohn1911 Please migrate the unit tests to register/unregister the mock `MockGroupItemSelectorProvider` using the bundle context instead of mocking and setting the service tracker map -- especially when the service tracker map field is final now.